relay/nakama/persona/persona.go

```diff
@@ -103,51 +188,32 @@ func CardinalCreatePersona(
 	return createPersonaResponse.TxHash, createPersonaResponse.Tick, nil
 }
 
-func cardinalQueryPersonaSigner(
+func ShowPersona(
 	ctx context.Context,
-	personaTag string,
-	tick uint64,
-	cardinalAddr string,
-) (signerAddress string, err error) {
-	readPersonaRequest := struct {
-		PersonaTag string `json:"personaTag"`
-		Tick       uint64 `json:"tick"`
-	}{
-		PersonaTag: personaTag,
-		Tick:       tick,
-	}
-
-	buf, err := json.Marshal(readPersonaRequest)
-	if err != nil {
-		return "", eris.Wrap(err, "")
-	}
-	httpReq, err := http.NewRequestWithContext(
-		ctx,
-		http.MethodPost,
-		utils.MakeHTTPURL(readPersonaSignerEndpoint, cardinalAddr),
-		bytes.NewReader(buf),
-	)
+	nk runtime.NakamaModule,
+	globalCardinalAddress string,
+) (res *StorageObj, err error) {
+	personaStorageObj, err := LoadPersonaTagStorageObj(ctx, nk)
 	if err != nil {
-		return "", eris.Wrap(err, "")
+		if eris.Is(eris.Cause(err), ErrPersonaTagStorageObjNotFound) {
```

Comment:
Is this extra if needed? It looks like the handler is checking for this error so it can return NotFound, but it should still find the error even if it's wrapped.

Reply:
you're right, killed it